src/manifest.json

```diff
@@ -3,7 +3,7 @@
 
   "name": "FTLabs GMail Signatures",
   "description": "A Chrome Extension that automagically adds links from RSS feeds to the end of emails sent in the Gmail web client",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
This seems like the correct version number. I'd remove the one in `package.json`